code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -1042,7 +1042,12 @@ func (sc *StorageSmartContract) commitBlobberConnection(
 		b.SavedData += changeSize
 		return nil
 	})
-	alloc.Stats.UsedSize += int64(float64(changeSize) * float64(alloc.DataShards) / float64(alloc.DataShards+alloc.ParityShards))
+	allocationWmSize := int64(float64(changeSize) * float64(alloc.DataShards) / float64(alloc.DataShards+alloc.ParityShards))
```

Comment:
we need to cover it with activator, since in general it can be now negative

Reply:
it's already under activator.